progress/internal/linear-progress.ts

```diff
@@ -16,8 +16,9 @@ import {Progress} from './progress.js';
 export class LinearProgress extends Progress {
   /**
    * Buffer amount to display, a fraction between 0 and `max`.
+   * If negative, the buffer is not displayed.
```

Comment:
Hm, actually what about using `0` as the default value to indicate that there is no buffer? I think that makes more better sense than using negative to indicate it.

If there's 0 buffer after all, there is no buffer.

Reply:
@asyncLiz thanks, the latest commit should address the above suggestions